portfolio/src/main/webapp/comments-script.js

```diff
@@ -1,3 +1,26 @@
+
+var credentials = {};
+
+function initiateFunctions(){
+    fetch("/auth")
+    .then(res => res.json())
+    .then(res => {
+        credentials = res;
+        if(res.loggedIn === true){
+            document.getElementById("commentForm").style.visibility = "visible";
```

Comment:
Another small suggestion, move visibility setting for commentForm to callFetch() as they belong to the same "comment" section.

You might want to rename "callFetch" to "loadComment"